pkg/lib/oauth/oidc/id_token.go

```diff
@@ -169,9 +169,10 @@ func (ti *IDTokenIssuer) IssueIDToken(client *config.OAuthClientConfig, s sessio
 	_ = claims.Set(jwt.IssuedAtKey, now.Unix())
 	_ = claims.Set(jwt.ExpirationKey, now.Add(IDTokenValidDuration).Unix())
 
-	for key, value := range s.SessionAttrs().Claims {
-		_ = claims.Set(string(key), value)
-	}
+	// Avoid putting user claims in ID token
```

Comment:
Can we also add a comment in the method `LoadUserClaims` to warn us not to add sensitive claims in the future?